linux/user-and-file-management/bash-environment/linux-shell-variables.md

```diff
@@ -0,0 +1,86 @@
+---
+author: kapnobatai136
+
+aspects:
+  - introduction
+
+type: normal
+
+category: must-know
+
+tags:
+  - introduction
+  - linux
+  - variables
+
+---
+
+# Variables
+
+---
+## Content
+
+Think of *variables* as a box. 
+
+They are used to store information. To make them easily identifiable, you put a label on them. 
+
+Let's try creating a variable now:
+
+```bash
+best_app="Enki"
+```
+
+> **Note:** don't add any spaces around the `=` sign or you'll get an error saying `"Command not found"`.
+
+We've taken the `"Enki"` information, and stored it in our `best_app` box:
+
+![variable-mental-model](https://img.enkipro.com/fa537341f3027f1cea7b76ecc3398e9d.png)
+
+To print this variable, you'd type:
+
+```bash
+echo $best_app
+Enki
+```
+
+> **Note:** when you are referencing variables, you need to prefix a `$` sign to the variable name. Otherwise, you are only printing the name of the variable.
```

Comment:
We use `$` to extract the value from the name. If we omit the `$`, we don't have just the name cause technically all we have is a string.

```suggestion
> **Note:** You need to prefix a `$` sign to the variable name to get its value
```

Also, IMHO, all of these should use an emoji:

```suggestion
> 💡You need to prefix a `$` sign to the variable name to get its value
```